devtools/conda-envs/psi.yaml

```diff
@@ -4,6 +4,7 @@ channels:
   - conda-forge
 dependencies:
   - psi4=1.5
+  - libint2=*=hc9558a2_9
```

Comment:
Is this tying to a *very specific* build of `libint2` on the `psi4` channel? If so, can we do a version instead?

Reply:
Yes it is, and sadly, no, it's build conditions, not source versions that matter. Different conditions work with psi through time (https://github.com/psi4/psi4/issues/2442). Psi started pinning build to L2 this spring, so v1.5 is really the most affected.